src/AzureAuth.Test/PublicClientAuthTest.cs

```diff
@@ -107,14 +107,38 @@ public void Good_Auth_Returns_TokenResult()
             // separately in AuthFlowResultExtensionsTest for converting an AuthFlowResult to EventData.
             this.telemetryService.SetupSequence(t => t.SendEvent("authflow_TestAuthFlow", It.IsAny<EventData>()));
 
-            // Act
-            TokenResult subject = this.Subject().Token(this.client, this.tenant, this.scopes, new[] { AuthMode.Web, AuthMode.DeviceCode }, this.domain, this.prompt, this.timeout);
+            EventData eventData = new EventData();
 
-            // Assert
+            // Act
+            TokenResult subject = this.Subject().Token(
+                this.client,
+                this.tenant,
+                this.scopes,
+                new[] { AuthMode.Web, AuthMode.DeviceCode },
+                this.domain,
+                this.prompt,
+                this.timeout,
+                eventData);
+
+            // Asserts
             subject.Should().Be(expected);
+
             // These logs are only those generated by this class/test.
             // What we do not see here are the debug and trace logs generated by the services that are mocked for the test.
-            this.logTarget.Logs.Should().BeEmpty();
+            this.logTarget.Logs.Should().ContainInOrder("Acquired AAD AT via TestAuthFlow in 0.00 sec");

```

Comment:
I see multiple types of log statements in `PublicClientAuth` . We should perhaps have tests that asserts logging in case of `authflow == null`

Reply:
Yes! These logs specifically have no test coverage right now and are moved in this PR from CommandToken to PublicClientAuth. A follow up PR (today hopefully) will add another integration test for the auth failed, and auth has errors test cases, where these will be exercised.